packages/ruleset/test/description-mentions-json.test.js

```diff
@@ -107,7 +107,20 @@ describe('Spectral rule: description-mentions-json', () => {
 
       testDocument.paths['/v1/movies'].post.responses['400'].content[
         'application/json'
-      ].schema.description = 'A JSON object containing the error details.';
+      ].schema = {
```

Comment:
Had to adjust this testcase due to the changes made to root-document.js (which were needed to get a clean validation run).